### PR TITLE
Forms

### PR DESCRIPTION
**Pull request by Fadahunsiseyi**
### In this MileStone, I completed the following;

-  Create a method called current_user in your ApplicationController to make current user data available to all controllers.

It will return the first user from the database.

- Don't worry about proper authentication at this time. You will add it in the upcoming projects.

- Create forms to perform the following functions:

- Creates a Post on behalf of the current_user (use the method that you created in your ApplicationController).

- Create a comment on behalf of the current_user (use the method that you created in your ApplicationController).

- Allow Users to add likes to Posts.

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+class CommentsController < ApplicationController
```
Comment (rloterh) [inline app/controllers/comments_controller.rb:1]: - [ ] `new` action is missing in your comments controller. Kindly note that this activity requires that you explicitly declare both `new` and `create` actions for comments controller. Kindly ensure to improve upon the same, which should be similar to:
```ruby
# controllers/comments_controller.rb

class CommentsController < ApplicationController
  ...

  def new
    @comment = Comment.new
  end

  def create
  
    ...
  end


  private 
  def comment_params
    params.require(:comment).permit(:text)
  end

	...
end

File: app/views/comments/index.html.erb [hunk 2]
```diff
@@ -0,0 +1 @@
+<h1>Comments#create</h1>
```
Comment (rloterh) [inline app/views/comments/index.html.erb]: - [ ] There should be a form in the `views/comments/new.html.erb` file which corresponds to the `new` action in _comments controller_, that accepts input for creating comments. You currently don't have this required _view_ file. Kindly ensure to implement the same.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Fadahunsi, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Highlights 👍  
- Good job with form implementation so far
- Good ReadMe

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

**You can also consider:**
- MIT license link in your ReadMe is broken, kindly endeavor to fix it.
Implementing the above change(s) will make your ReadMe/ repository stand out even better 😉

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/comments/index.html.erb (hunk 2 above)
Comment (Fadahunsiseyi) [reply app/views/comments/index.html.erb]: @rloterh  Thank you, I have implemented the required changes.

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @Fadahunsiseyi,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**